chain/ethereum/src/ethereum_adapter.rs

```diff
@@ -306,17 +301,18 @@ where
             } else {
                 debug!(logger, "Requesting traces for blocks [{}, {}]", start, end);
             }
-            Some(
+            Some(futures::future::ok((
                 eth.traces(
                     &logger,
                     subgraph_metrics.clone(),
                     start,
                     end,
                     addresses.clone(),
-                )
-                .map(move |traces| (traces, new_start)),
-            )
+                ),
+                new_start,
+            )))
         })
+        .buffered(*BLOCK_BATCH_SIZE)
```

Comment:
We should probably also update the documentation in `docs/environment-variables.md` to indicate that `ETHEREUM_BLOCK_BATCH_SIZE` is used to control the degree of parallelism of various operations, not just requests for blocks (and that it defaults to 10)

Reply:
I had forgotten to do this, thanks!